pangeo-binder/values.yaml

```diff
@@ -189,19 +193,39 @@ kube-lego:
 
 dask-gateway:
   gateway:
-    clusterManager:
-      image:
-        name: daskgateway/dask-gateway-server
-        tag: 0.7.0
-        pullPolicy: IfNotPresent
-      clusterStartTimeout: 600
-      workerStartTimeout: 600
+    prefix: "/services/dask-gateway"
+    image:
+      name: daskgateway/dask-gateway-server
+      tag: 0.7.1
+      pullPolicy: IfNotPresent
+
+    backend:
+      scheduler:
+        cores:
+          request: 0.8
+          limit: 1
+        memory:
+          request: 1G
+          limit: 2G
+        extraPodConfig:
+#          name: scheduler-{JUPYTERHUB_USER}-{uuid}
```

Comment:
Would be nice to include user name in scheduler pod name (see comment on worker pods below)

Reply:
usernames can include invalid characters or be too long to use in k8s object names. If you're trying to see active user clusters, you can query the `daskclusters` objects:

```
$ kubectl get daskclusters
```

You can also configure user-specific annotations (annotations don't have as strict restrictions on their values), which would let you put the username annotation on user pods:

```python
from dask_gateway_server.options import Options

def cluster_options(user):
    """Given a user, return a set of cluster options (with a handler) for that user.

    You can use this to add user-specific functionality"""

    def handler(options):
        # Add some annotation (key can be whatever) with the username
        annotations = {"jupyterhub.org/username": user.name}
        return {
            "scheduler_extra_pod_annotations": annotations,
            "worker_extra_pod_annotations": annotations,
        }
    
    return Options(handler=handler)

c.KubeBackend.cluster_options = cluster_options
```